scripts/wordlist.txt

```diff
@@ -144,6 +144,8 @@ Skoda
 SLYR
 SSL
 Survey123
+synchronization
```

Comment:
We use UK spelling, so it should be  **synchronisation**. Please remove this from the wordlist so that the spellcheck works correctly.